metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractExecutablePlugin.java

```diff
@@ -226,6 +228,8 @@ public MonitorResult monitor(DpsClient dpsClient) throws ExternalTaskException {
     TaskInfo taskInfo;
     try {
       taskInfo = dpsClient.getTaskProgress(getTopologyName(), Long.parseLong(getExternalTaskId()));
+    }catch (AccessDeniedOrObjectDoesNotExistException e){
```

Comment:
watch out for the formatting apply, "format code" as mention above also to this file.

Reply:
update `ExecutablePlugin.java` to support the exception and update the java documentation.
`  MonitorResult monitor(DpsClient dpsClient) throws ExternalTaskException, UnrecoverableExternalTaskException;`

update method in `AbstractExecutablePlugin.java` to support the exception.
`@Override
  public MonitorResult monitor(DpsClient dpsClient) throws ExternalTaskException, UnrecoverableExternalTaskException {`